benches/water.rs

```diff
@@ -41,9 +41,10 @@ fn energy_ewald(bencher: &mut Bencher) {
 fn forces_ewald(bencher: &mut Bencher) {
     let system = utils::get_system("water");
     let ewald = get_ewald();
+    let mut forces = vec![Vector3D::zero(); system.size()];
```

Comment:
I put the forces initialization out of the loop to avoid benchmarking it, is that a correct way to do it ?